frontend/rolecall/src/app/api/login_api.service.spec.ts

```diff
@@ -80,8 +79,8 @@ describe('LoginApi', () => {
     await loginRespProm;
     mockObj.isSignedInVal = false;
 
-    expect(isNullOrUndefined(service.authInstance.currentUser.get()))
-        .toBeTrue();
+    expect(service?.authInstance?.currentUser?.get())
+        ?.toBeTrue();
```

Comment:
I think this is supposed to check that the result is null, so possibly you want something like this:

```
expect(service.authInstance.currentUser.get()).toBeNull();
```

If the intermediate steps are undefined, then the test will also fail correctly.

Reply:
Changed.